docs/3.0.0-beta.x/plugins/email.md

```diff
@@ -111,3 +111,42 @@ If you want to create your own provider without publishing it on **npm** you can
 ```
 
 - Finally, run `yarn install` or `npm install` to install your new custom provider.
+
+## Trouble shooting
+
+You receive an `Auth.form.error.email.invalid` error even if the email is valid and exist in your database.
+
+Here is the error response you get from the API.
+
+```json
+{
+  "statusCode": 400,
+  "error": "Bad Request",
+  "message": [
+    {
+      "messages": [
+        {
+          "id": "Auth.form.error.email.invalid"
+        }
+      ]
+    }
+  ]
+}
+```
+
+It's due to your IP connection. By default, Strapi use the [`sendmail`](https://github.com/guileen/node-sendmail) node module.
```

Comment:
This error is due to your IP connection. By default, Strapi uses the [`sendmail`] (https://github.com/guileen/node-sendmail) package.